build-tools/build-indexes

```diff
@@ -461,6 +461,14 @@ process.stdout.write("Building `data/teambuilder-tables.js`... ");
 				if (genNum === 2) goodItems.push(id);
 				else greatItems.push(id);
 				break;
+			case 'aguavberry':
+			case 'figyberry':
+			case 'iapapaberry':
+			case 'magoberry':
+			case 'wikiberry':
+				if (genNum >= 7) goodItems.push(id);
```

Comment:
Why is it merely a good item? I think it would be better to make it a great item, and move Sitrus Berry to outclassed items in Gen7 only.

Reply:
Sitrus Berry activates at 50% and the others at 25%, right? So it would still have uses.

Reply:
Iirc teambuilder usefulness is not based on Doubles/VGC play, but the super Sitrus (Aguav, etc.) Berries are pivotal parts of both tiers, with regular Sitrus Berry having more niche use, but still being a viable choice.

Reply:
I suppose I just didn't want to clutter the Great Items list with 5 near-identical berries. So do we put Aguav etc at Great, Sitrus at good?

Reply:
Oh right, forgot the difference of activation level.